server/tests/controllers/test_contributors_controller.py

```diff
@@ -0,0 +1,75 @@
+import json
+
+import pytest
+
+
+@pytest.mark.parametrize("cached", [False, True], ids=["no cache", "with cache"])
+async def test_get_contributors_as_admin(client, cached, headers, app, cache):
+    await _test_get_contributors(client, cached, headers, app, cache)
+
+
+@pytest.mark.parametrize("cached", [False, True], ids=["no cache", "with cache"])
+async def test_get_contributors_as_non_admin(client, cached, headers, app, cache, eiso):
+    await _test_get_contributors(client, cached, headers, app, cache)
+
+
+async def _test_get_contributors(client, cached, headers, app, cache):
+    if cached:
+        app._cache = cache
+
+    response = await client.request(
+        method="GET", path="/v1/contributors/1", headers=headers,
+    )
+
+    assert response.status == 200
+
+    contribs = json.loads((await response.read()).decode("utf-8"))
+
+    assert len(contribs) == 206
+    assert len(set(c["login"] for c in contribs)) == len(contribs)
```

Comment:
Can you please use `Contributor.from_dict()` here instead of dictionary lookups so that we test the validity of that model.